packages/components/src/internal/util/Date.ts

```diff
@@ -589,14 +602,20 @@ export function formatDateTime(date: Date | string | number, timezone?: string,
     return _formatDate(date, dateFormat ?? getDateFNSDateTimeFormat(), timezone);
 }
 
+export function formatTime(timeStr: string, timeFormat?: string): string {
```

Comment:
This is inconsistent with `formatDate` and `formatDateTime` in that it returns the value of `timeStr` when it cannot parse the date. To be consistent with those I would expect the following logic:

```ts
export function formatTime(timeStr: string, timeFormat?: string): string {
    const date = parseTime(timeStr);
    if (!date) return undefined;

    return format(date, timeFormat ?? getDateFNSTimeFormat());
}
```

I'm not sure how I feel about this. On one hand it allows passing through invalid values (as your tests clearly cover), on the other, it will only return valid formatted times. Either way, I feel like we should be consistent.

Reply:
I made it consistent with the other utils to return undefined.

Reply:
It doesn't look like it quite is. It still returns `timeStr` when `!timeStr` which could return `''` or `null`, etc. Consider:

```ts
export function formatTime(timeStr: string, timeFormat?: string): string {
    const date = parseTime(timeStr);
    if (!date) return undefined;

    return format(date, timeFormat ?? getDateFNSTimeFormat());
}
```